### PR TITLE
Small optimization for getTeamIDs

### PR DESCRIPTION
**Pull request by lunny**
The latest XORM has supported only Find IDs. So let's use the new method.

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

File: models/org.go [hunk 1]
```diff
@@ -563,18 +563,22 @@ func (org *User) getUserTeams(e Engine, userID int64, cols ...string) ([]*Team,
 		Find(&teams)
 }
 
+func (org *User) getOrgTeamIDs(e Engine, userID int64) ([]int64, error) {
```
Comment (ethantkoenig) [inline models/org.go]: Should this be `getUserTeamIDs` (to match `GetUserTeamIDs`)?

File: models/org.go [hunk 2]
```diff
@@ -563,18 +563,22 @@ func (org *User) getUserTeams(e Engine, userID int64, cols ...string) ([]*Team,
...
+	teamIDs := make([]int64, 0, org.NumTeams)
+	return teamIDs, e.
+		Table("team").
+		Cols("team.id").
+		Where("`team_user`.org_id = ?", org.ID).
+		Join("INNER", "team_user", "`team_user`.team_id = team.id").
+		Join("INNER", "user", "`user`.id=team_user.uid").
+		And("`team_user`.uid = ?", userID).
+		Asc("`user`.name").
```
Comment (ethantkoenig) [inline models/org.go]: Why is this `ORDER BY` necessary? We are only looking for one user (i.e. the user with ID `userID`)

Plus, if we get rid of the `ORDER BY`, we won't need to join with the `user` table :smile:

**Review by ethantkoenig — CHANGES_REQUESTED** [review]

File: models/org.go (hunk 1 above)
Comment (lunny) [reply models/org.go]: done.

File: models/org.go (hunk 2 above)
Comment (lunny) [reply models/org.go]: done.

**Review by ethantkoenig — APPROVED** [review]

**Comment by ethantkoenig** [comment]
LGTM

**Pull request merged**